src/Ombi/ClientApp/src/app/settings/plex/plex.component.scss

```
@@ -1,300 +1,692 @@
+            background: rgba(255, 255, 255, 0.05);
+            color: $plex-text-muted;
+        }
+    }
+
+    &__name {
+        grid-area: name;
+        font-size: 15px;
+        font-weight: 600;
+        color: #ffffff;
+        line-height: 1.2;
+        word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)